pueue/src/bin/pueue.nu

```diff
@@ -0,0 +1,191 @@
+module pueue_completions {
```

Comment:
Could you also add a small comment, that this file is hand-written and needs to be updated whenever there're any changes to the cli interface :)?